tools/functional-tester/etcd-runner/main.go

```diff
@@ -227,7 +227,7 @@ func runWatcher(eps []string) {
 	keyPrePrefix := 30              // max number of keyPrePrefixs for put operation
 
 	prefixes := generateUniqueKeys(5, noOfPrefixes)
-	keys := generateRandomKeys(10, keyPrePrefix)
+	keys := generateUniqueKeys(10, keyPrePrefix)
```

Comment:
@sinsharat Actually if we do rev comparison around line 270 instead of doing existence testing, we can use random keys. No?